terraform/open-vpn/r53.tf

```diff
@@ -0,0 +1,15 @@
+resource "aws_route53_zone" "myzone" {
```

Comment:
R53 zone probably needs to be at the higher level or separate module to set it up.
Think it would be unlikely to have a zone just for pihole and vpn?

Reply:
probably .. will raise new Feature request to pull route53 out into another module